### PR TITLE
zh_trans: docs/reference/glossary/quantity.md

### PR DESCRIPTION
**Pull request by Mr-Linus**

### CODE REVIEW

**Comment by chenrui333** [comment]
/tide/squash

**Comment by idealhack** [comment]
`tide/squash` can only be added by a human. It tells tide to squash commits when merging a PR.

I found it’s not needed in this repo since tide has been configured to do so.

As long as we’re running commands to merge PRs, we don’t have to ask authors to squash their commits nor adding this label.

cc @markthink

**Comment by chenrui333** [comment]
@idealhack that is very good to know!

File: content/zh/docs/reference/glossary/quantity.md [hunk 1]
```diff
@@ -10,10 +23,25 @@ aka:
 tags:
 - core-object
 ---
- A whole-number representation of small or large numbers using SI suffixes.
+-->
 
 <!--more-->
 
+数量是使用紧凑的整数表示法的小数或大数的表示，并带有 SI 后缀。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/quantity.md]: SI 翻译成“国际计量单位制（SI）”

**Comment by Mr-Linus** [comment]
@tengqm 感谢指点，已修正

**Pull request merged**